packages/main/src/DatePicker.ts

```diff
@@ -600,7 +600,8 @@ class DatePicker extends DateComponentBase implements IFormElement {
 
 	_click(e: MouseEvent) {
 		if (isPhone()) {
-			this.responsivePopover!.showAt(this);
```

Comment:
Hello colleagues,

Please do have in mind that we're already adapting the ui5-date-picker, ui5-datetime-picker and other date components to the open/opener API via the following PR: https://github.com/SAP/ui5-webcomponents/pull/8749
We're currently quite ahead with the implementation and I would suggest not to merge this PR before we merge ours.
That way nothing will break due to the Popup.showAt method removal.